test/framework/src/main/java/org/elasticsearch/test/junit/listeners/ReproduceInfoPrinter.java

```diff
@@ -149,6 +149,7 @@ public ReproduceErrorMessageBuilder appendESProperties() {
             }
             appendOpt("tests.locale", Locale.getDefault().toLanguageTag());
             appendOpt("tests.timezone", TimeZone.getDefault().getID());
+            appendOpt("tests.distribution", System.getProperty("tests.distribution"));
```

Comment:
If you override the distribution on the command line then it is important to override it again. If you *don't* override it on the command line then setting this environment variable does nothing which is fine.